app/src/main/res/values/strings.xml

```diff
@@ -17,4 +17,11 @@
     <string name="fragment_request_post_id">Post id</string>
     <string name="fragment_request_invalid_input">Invalid input, please enter a valid id.</string>
 
+    <!-- Login Activity -->
```

Comment:
This comment to separate the different resources is OK but the name should not refer to a specific activity or fragment. Just refer to what part of the app is using it. In this case, is just Login. After that, all the resources should begin with the word or words as the comment. For example:
```
<!-- Login -->
    <string name="login_email_hint">Email</string>
    <string name="login_password_hint">Password</string>
    <string name="login_main_button">Log In</string>
    <string name="login_sign_up_button">Sign Up</string>
    <string name="login_terms_conditions_text">By joining you agree to our </string>
    <string name="login_terms_conditions_link"><u>Terms &amp; Conditions</u></string>

```